Sources/JExtractSwiftLib/JNI/JNISwift2JavaGenerator+JavaBindingsPrinting.swift

```diff
@@ -206,13 +206,15 @@ extension JNISwift2JavaGenerator {
     printDeclDocumentation(&printer, decl)
     printer.printBraceBlock("public \(renderFunctionSignature(decl))") { printer in
       var arguments = translatedDecl.translatedFunctionSignature.parameters.map(\.name)
-      arguments.append("selfPointer")
+
+      let selfVarName = "self$"
+      arguments.append(selfVarName)
 
       let returnKeyword = translatedDecl.translatedFunctionSignature.resultType.isVoid ? "" : "return "
 
       printer.print(
         """
-        long selfPointer = this.pointer();
+        long \(selfVarName) = this.$memoryAddress();
```

Comment:
These are basically unsafe, so I figured we shoudl hide them using our $ pattern.